packages/@sanity/core/src/commands/graphql/deleteGraphQLAPICommand.js

```diff
@@ -0,0 +1,9 @@
+const lazyRequire = require('@sanity/util/lib/lazyRequire')
+
+export default {
+  name: 'delete',
```

Comment:
Did we consider `sanity graphql undeploy`? Since we went for `sanity undeploy` for content studio, I feel it's a bit more natural to name it undeploy here as well.

Reply:
True.